packages/fetch-and-compile/package.json

```diff
@@ -46,6 +47,14 @@
   "devDependencies": {
     "@truffle/compile-common": "^0.7.22",
     "@truffle/config": "^1.3.12",
+    "@types/chai": "^4.2.22",
+    "@types/debug": "^4.1.7",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.11.6",
+    "@types/semver": "^7.3.9",
```

Comment:
Hey @leeftk, still looking for an answer to your question.  It looks like you're going with just leaving them mismatched for now, which isn't a problem, but it would be good to explicitly note that you are making that decision, and, ideally, why.

Reply:
Hey yea sorry I didn't answer here I guess I spoke with Amal about it and just forgot. It seems like there's different versions for these packages on NPM so for `@types/debug'  the latest version is 4.1.7. While the latest version of `debug' is 4.3.2. And this is the case for the other mismatches as well. So I could downgrade `debug` I guess but since it builds like you said above I figured I would just leave them. I'm open to any suggestion though so let me know what you think is best. Thanks :).

Reply:
OK, but I was asking about `semver`, not `debug`?

Reply:
Oh, OK, that makes sense then.  Thanks for clearing that up!